api/working-with-extensions/publishing-extension.md

```diff
@@ -202,7 +202,7 @@ Also see [Marketplace Presentation Tips](/api/references/extension-manifest#mark
 
 ### Verify a publisher
 
-You can become a **verified publisher** by verifying ownership of an eligible domain associated with your brand or identity.
+You can become a **verified publisher** by verifying ownership of a [eligible domain](#eligible-domains) associated with your brand or identity.
```

Comment:
I am not an English native speaker but I thought the rule is that `an` goes before a syllable and `a` otherwise? Was I wrong all these years :)

Reply:
```suggestion
You can become a **verified publisher** by verifying ownership of an [eligible domain](#eligible-domains) associated with your brand or identity.
```

Reply:
done 👍